distributor-api-v1/use-cases/on-session-payments.md

```diff
@@ -0,0 +1,30 @@
+# On session payments
```

Comment:
We should add a link to this file and to to `distributor-api-v1/use-cases/README.md`  `https://github.com/MewsSystems/gitbook-distributor-guide/blob/master/SUMMARY.md` so it's visible in gitbook menu